### PR TITLE
Update schoology extension

### PR DESCRIPTION
**Pull request by Visual-Studio-Coder**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/schoology/.gitignore [hunk 1]
```diff
@@ -1,10 +0,0 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
-# dependencies
-/node_modules
-
-# Raycast specific files
-raycast-env.d.ts
-
-# misc
-.DS_Store
```
Comment (pernielsentikaer) [inline extensions/schoology/.gitignore:10]: Why did you delete this file? Sounds like a keeper to me 🙂

Comment (Visual-Studio-Coder) [reply extensions/schoology/.gitignore:10]: Idk. I deleted it because github was giving me issues.

**Comment by Visual-Studio-Coder** [comment]
Can I ask what's missing? Is it the gitignore or something?

**Comment by pernielsentikaer** [comment]
Would be a good idea to keep it, you can always add in the next release
